src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/NodeStepNamePredicate.java

```diff
@@ -41,7 +41,7 @@
 import java.util.logging.Logger;
 
 /** Predicate that matches {@link FlowNode}s (specifically {@link StepNode}s) with a specific {@link StepDescriptor} name.
- *  May be used in preference to {@link NodeStepTypePredicate} in cases whern dependencie structures prevent import
+ *  May be used in preference to {@link NodeStepTypePredicate} in cases whern dependency structures prevent import
```

Comment:
...I'll admit it, I'm entertained that you fixed `dependencie` but not `whern`. =)

Reply:
I kind of like picturing the javadocs written by a crusty old gold-miner

Reply:
... or for some reason "dependencie" grated enough to be noticeable but "whern" didn't